test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -16,14 +16,44 @@ export default class PossibleIssues extends Component {
     }
 
     async fetchIssues() {
-        const { buildName, testName } = getParams( this.props.location.search );
+        const { testId, buildName, testName } = getParams( this.props.location.search );
+        const generalTestName = testName.replace(/_\d+$/, '');
 
+        // fetch test output content
+        const fetchData = await fetch( `/api/getTestById?id=${testId} `, {
+            method: 'get'
+        } );
+        const info = await fetchData.json();
+        const fetchTest = await fetch( `/api/getOutputById?id=${info.testOutputId}`, {
+            method: 'get'
+        } );
+        const result = await fetchTest.json();
+        const testOutput = result.output;
+
+        // query ML Server for possible issues
+        let mlIssue = '', mlIssueRepo = '', mlIssueNum = '';
+        const mlResponse = await fetch( 'http://127.0.0.1:5000/predict', {
```

Comment:
I do not think we should use `127.0.0.1` (localhost). Local/docker will fail. Please use a real URL.

Reply:
Sure, I will test with TRSS server to replace it with the actual URL. Thanks.

Reply:
Created a new issue for the production server deployment, https://github.com/adoptium/aqa-test-tools/issues/408, Thanks.

Reply:
The url here has been updated to `https://ml.trss.adoptopenjdk.net/predict`, and temporarily all TRSS servers will use this ml server (tested on [fyre vm](http://ml.jobbing1.fyre.ibm.com/), will set it up on Adopt TRSS machine after this PR merged). Thanks.